tests/test_kubernetes_tools.py

```diff
@@ -275,6 +276,28 @@ def test_get_kubernetes_environment(self):
         ret = self.deployment.get_kubernetes_environment()
         assert 'PAASTA_POD_IP' in [env.name for env in ret]
 
+    def test_get_resource_requirements(self):
+        with mock.patch(
+            'paasta_tools.kubernetes_tools.KubernetesDeploymentConfig.get_cpus', autospec=True,
+            return_value=0.3,
+        ), mock.patch(
+            'paasta_tools.kubernetes_tools.KubernetesDeploymentConfig.get_cpu_burst_pct', autospec=True,
+            return_value=200,
+        ), mock.patch(
+            'paasta_tools.kubernetes_tools.KubernetesDeploymentConfig.get_mem', autospec=True,
+            return_value=2048,
+        ):
+            assert self.deployment.get_resource_requirements() == V1ResourceRequirements(
+                limits={
+                    'cpu': 0.6,
+                    'memory': 2048,
```

Comment:
Please see the comment above, this one and the one below probably should be `str(2048*1024)`.